src/templates/string-array-nodes/string-array-calls-wrapper/SelfDefendingTemplate.ts

```diff
@@ -1,46 +1,59 @@
 import { IEscapeSequenceEncoder } from '../../../interfaces/utils/IEscapeSequenceEncoder';
+import { Utils } from '../../../utils/Utils';
```

Comment:
Check this line as example how to pass argument to template. In the example below `escapeSequenceEncoder` is passed to the template.
https://github.com/javascript-obfuscator/javascript-obfuscator/blob/master/src/custom-nodes/self-defending-nodes/SelfDefendingUnicodeNode.ts#L73

And then in the template you can get it in the parameter:
https://github.com/javascript-obfuscator/javascript-obfuscator/blob/master/src/templates/self-defending-nodes/self-defending-unicode-node/SelfDefendingTemplate.ts#L10